dbt_platform_helper/commands/pipeline.py

```diff
@@ -32,13 +37,13 @@ def pipeline():
 
 @pipeline.command()
 def generate():
-    """Given a pipelines.yml file, generate environment and service deployment
-    pipelines."""
+    """Given a platform-config.yml file, generate environment and service
+    deployment pipelines."""
     templates = setup_templates()
 
     app_name = get_application_name()
 
-    pipeline_config = _safe_load_config("pipelines.yml", PIPELINES_SCHEMA)
+    pipeline_config = _safe_load_config("platform-config.yml", PLATFORM_CONFIG_SCHEMA)
```

Comment:
Should we use `PLATFORM_CONFIG_FILE` in all these places where it's using `platform-config.yml`?